bin/garden

```diff
@@ -203,7 +203,8 @@ class GardenTool(object):
             index += 1
             data += buf
             count += len(buf)
-            print('Progression', count, animation[index % len(animation)], '\r')
+            print(' '.join([str(x) for x in ('Progression', count,
+                            animation[index % len(animation)], '\r')]))
             sys.stdout.flush()
```

Comment:
This is looking a bit complicated. `'{} {} {} {}'.format('Progression', count, animation[index % len(animation)], '\r')` is probably better.